docs/products/flink/howto/real_time_alerting_solution_cli.rst

```diff
@@ -0,0 +1,392 @@
+Create a real-time alerting solution - Aiven CLI
+================================================
+
+This Aiven CLI tutorial shows you an example of how to combine Aiven for Apache Flink with Aiven for Apache Kafka and Aiven for PostgreSQL services to create a solution that provides real-time alerting data for CPU loads.
+
+Architecture overview
+---------------------
+
+This example involves creating an Apache Kafka source topic that provides a stream of metrics data, a PostgreSQL database that contains data on the alerting thresholds, and an Apache Flink service that combines these two services and pushes the filtered data to a separate Apache Kafka topic or PostgreSQL table.
+
+.. mermaid::
+
+    graph LR;
+
+        id1(Kafka)-- metrics stream -->id3(Flink);
+        id2(PostgreSQL)-- threshold data -->id3;
+        id3-. filtered data .->id4(Kafka);
+        id3-. filtered data .->id5(PostgreSQL);
+
+The article includes the steps that you need when using the `Aiven CLI <https://github.com/aiven/aiven-client>`_ along with a few different samples of how you can set thresholds for alerts. For connecting to your PostgreSQL service, this example uses the Aiven CLI calling `psql <https://www.postgresql.org/docs/current/app-psql.html>`_, but you can also use other tools if you prefer.
+
+In addition, the instructions show you how to use a separate Python tool, `Apache Kafka Python fake data producer <https://github.com/aiven/python-fake-data-producer-for-apache-kafka>`_, to create sample records for your Apache Kafka topic that provides the streamed data.
```

Comment:
replace the link to point to the docker version https://github.com/aiven/fake-data-producer-for-apache-kafka-docker